docs/reference/quality-model-documents/open-source-health.md

```diff
@@ -0,0 +1,198 @@
+SIG Open Source Health Quality Model
+=========================================
+
+### Guidance for producers
+December 7, 2023
+
+<sig-toc></sig-toc>
+
+# Introduction
+
+Contemporary software development heavily relies on open source
+third-party dependencies. Dependencies reduce the burden of spending
+precious resources to implement tedious but important features, with the
+benefit of relying on a wide community support that takes care of the
+maintenance effort.
+
+Nevertheless, using third-party open source dependencies also comes with
+challenges and risks: daily, new security issues (vulnerabilities) found
+in third-party packages might lead to exploits by malicious actors. The
+nature of open source software development also comes with the risks of
+lacking support, restrictive license usage, and more.
+
+Since 2021, SIG utilized the Open Source Health capability to assess the
+risks involved by using third-party open source dependencies. While the
+risk assessment provided precious initial indication to stakeholders, it
+did not provide how such risks compare with the market. SIG’s Open
+Source Health Quality model has been developed to fill this gap,
+adopting similar benchmarking methodologies used in other SIG Quality
+models.
+
+This document outlines guidance and provides explanation to software
+producers about the measurement method of SIG applied to open source
+third-party dependencies used in software products. For the measurements
+SIG considers, the threshold measurement values are provided for scoring
+at the level of 4 stars.
+
+Throughout this document, the terms "package" and "dependency" refer to
+third-party open source dependency integrated and used by a software
+product.
+
+# Model Definitions
+
+The SIG Open Source Health Quality Model measurements are performed on
+third-party open source dependencies.
+
+Elements of a dependency that contribute to the measurements in the Open
+Source Health Quality model are:
+
+- Vulnerabilities
+- Dependency Freshness
+- License use
+- Activity
+- Management
+
+As such, for each of the above SIG has associated a property, that is
+captured by a metric. The rest of this document outlines the details and
+thresholds for each of them.
+
+# Guidance for producers
+
+For each measurement, the threshold values are provided that are
```

Comment:
"For each measurement, the values provided are the ones necessary to get a 4-star rating."